README.md

```diff
@@ -441,6 +443,33 @@ temporary file (outside the `source.d` and `features.d` directories), and,
 atomically create/update the original file by doing a filesystem move
 operation.
 
+## Extended resources [experimental]
+
+This feature is experimental and by no means a replacement for the usage of
+device plugins.
+
+Labels which have integer values, can be promoted to Kubernetes extended
+resources by listing them to the master `--resource-labels` command line flag.
+These labels won't then show in the node label section, they will appear only
+as extended resources.
+
+An example use-case for the extended resources could be based on a hook which
+creates a label for the node SGX EPC memory section size. By giving the name of
+that label in the `--resource-labels` flag, that value will then turn into an
+extended resource of the node, allowing PODs to request that resource and the
+Kubernetes scheduler to schedule such PODs to only those nodes which have a
+sufficient capacity of said resource left.
+
+Similar to labels, the default namespace `feature.node.kubernetes.io` is
+automatically prefixed to the extended resource, if the promoted label doesn't
+have a namespace. The default namespace is not stored in node annotations.
+
+Example usage of the command line arguments, using a new namespace:
+`nfd-master --resource-labels=sgx.some.ns/epc --extra-label-ns=sgx.some.ns`
```

Comment:
One final comment 😉 Should we add an example of a default ns label, in order to cover that use case, too. Smth like
```
nfd-master --resource-labels=my_source-my.feature,sgx.some.ns/epc --extra-label-ns=sgx.some.ns
```

Would produce extended resources
```
sgx.some.ns/epc: <label value>
feature.node.kubernetes.io/my_source-my.feature: <label value>
```